packages/@lwc/engine/src/framework/__tests__/events.spec.ts

```diff
@@ -6,6 +6,7 @@
  */
 import { compileTemplate } from 'test-utils';
 import { createElement, LightningElement } from '../main';
+import { registerDecorators } from '../decorators/register';
```

Comment:
We should be consistent and import `createElement`, `LightningElement`, and `registerDecorators` from the same path.